docs/4.0. Behaviour.md

```diff
@@ -27,8 +27,17 @@ Where a server implementation supports concurrent application of settings change
 # Scheduled Activations
 
 IS-05 provides a mechanism whereby the activation of staged transport parameters can be performed at a particular TAI time, or after a given interval.
+
 The primary use case for this behaviour is to allow the synchronisation of large salvo operations, where multiple pieces of equipment should carry out a change in transport parameters simultaneously. A controller may stage parameters with multiple IS-05 APIs with the same activation timestamp, and know that activation will occur at the same time on all devices regardless of the latency between the controller and the device. It is not intended to act as a mechanism for scheduling activations far in the future.
 
 In the event of an error occurring between scheduling an activation and the activation time, the IS-05 transport parameters should reflect the current state of the device. For example, if a sender is no longer sending data at all it should reflect this by setting `master_enable` to false.
 
 API clients should check back with the API after the expected activation time to ensure activations have completed successfully, and that the device is in the expected state. Note that a client may normally expect to see a change in the IS-04 version timestamp upon a successful update of transport parameters, and should monitor for this version timestamp update via the IS-04 web-socket where IS-05 is being used along-side IS-04.
+
+# Externally Defined Parameters
+
+IS-05 aims to provide a defined core set of parameters for each supported transport type. These parameters are based upon the standard(s) which define the transport type.
+
+In some cases it may be necessary to advertise and accept additional parameters in order to make a connection, for example where an external specification extends an existing transport type in order to fulfil its own requirements. Where this is necessary, IS-05 provides the capability to define additional transport paramaters which use the prefix `ext_`. These transport parameters are governed by the same constraints structure as any other parameter, but are intended to make IS-05 more extensible for these use cases.
+
+Where `ext_` parameters are utilised, their naming, usage and versioning is defined external to IS-05.
```

Comment:
I wonder if we need guidance on what a client should do if it encounters ext_ parameters in a transport type it wasn't expecting? I imagine the best behaviour is to just ignore them on the basis that whoever designed the ext_ parameter should of made it backward compatible if they intended the given client to use it?